docs/cn/overview/JobService.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: global
+title: 作业服务
+group: Overview
+priority: 3
+---
+
+* 内容列表
+{:toc}
+
+## 架构概览
+
+Alluxio 作业服务是负责将各种不同类型的操作分配给 Job Worker的任务调度框架。
+
+Master 负责将Job分配为更小的任务，供 Job Worker 执行并管理Job的完成状态。
+
+Job Worker 将来自 Job Master 的任务排列（queue)，并通过管理可配置的固定线程池(`alluxio.job.worker.threadpool.size`)来完成这些任务。
+
+## 不同类型的作业
+
+### 加载
+
+Load Job 作为`fs distributedLoad` CLI 命令的一部分，会按特定的副本数将文件加载到 Alluxio。
+
+### 迁移
+
+Migrate Job 作为`fs distributedCp`  和 `fs distributedMv` CLI 命令的一部分，负责使用固定的 [WriteType]({{ '/en/overview/Architecture.html#data-flow-write' | relativize_url }}) 进行数据复制/移动。
```

Comment:
'/en/overview/Architecture.html#data-flow-write' may need to be updated to the Chinese doc link. I think we will have it soon, maybe we can wait for the latest doc and then update the link.